Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -304,6 +304,27 @@ class UserScriptManager {
         
         return WKUserScript(source: alteredSource, injectionTime: .atDocumentStart, forMainFrameOnly: false)
     }()
+    
+    private let MediaBackgroundingScript: WKUserScript? = {
+        guard let path = Bundle.main.path(forResource: "MediaBackgrounding", ofType: "js"), let source = try? String(contentsOfFile: path) else {
+            log.error("Failed to load MediaBackgrounding.js")
+            return nil
+        }
+        
+        var alteredSource = source
+        let token = UserScriptManager.securityToken.uuidString.replacingOccurrences(of: "-", with: "", options: .literal)
+        
+        let replacements = [
```

Comment:
what is the purpose of doing these replacements? cc @jumde

Reply:
When you override a property or inject code, someone's page can then override or replace that function with its own implementation. So the solution is to randomly generate the name and make it non-enumerable so the website cannot see it or change it.

This is solved in iOS 14+ with `WKContentWorld` but we currently support iOS 13+.